src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,228 @@
+package duke;
+
+
+import java.util.ArrayList;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.io.FileWriter;
+import java.io.File;
+import java.io.IOException;
+
+public class Duke {
+    public static final String INITIALISING = "initialising";
+    public static final int MAX_LIST_SIZE = 100;
+    private static ArrayList<Task> tasks = new ArrayList<>(MAX_LIST_SIZE);
+    public static final String FILEPATH = "data/duke.txt";
+    public static final String DIRPATH = "data";
+    private static int listSize = 0;
+
+
+    public static void writeToFile() throws IOException {
+        FileWriter fw = new FileWriter(FILEPATH);
```

Comment:
Consider using a more descriptive variable name
```suggestion
        FileWriter filewriter = new FileWriter(FILEPATH);
```